pkg/syncer/syncer.go

```diff
@@ -100,14 +100,21 @@ func New(from, to *rest.Config, upsertFn UpsertFunc, deleteFn DeleteFunc, handle
 		namespace: os.Getenv(SyncerNamespaceKey),
 	}
 
-	fromClient := dynamic.NewForConfigOrDie(from)
-	fromDSIF := dynamicinformer.NewFilteredDynamicSharedInformerFactory(fromClient, resyncPeriod, metav1.NamespaceAll, func(o *metav1.ListOptions) {
+	fromClient, err := dynamic.NewClusterForConfig(from)
+	if err != nil {
+		return nil, err
+	}
+	fromDSIF := dynamicinformer.NewFilteredDynamicSharedInformerFactory(fromClient.Cluster(logicalClusterID), resyncPeriod, metav1.NamespaceAll, func(o *metav1.ListOptions) {
```

Comment:
Long-term, I'm guessing this will need to change so it's syncing against a "virtual view" of everything scheduled to "me" (my physical cluster or location), and not just from a single logical cluster?

Reply:
Yeah, I expect we will feed it a client capable of such a thing in the future